lib/workos/refresh_authentication_response.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+# typed: true
+
+module WorkOS
+  # The AuthenticationResponse class represents an Authentication Response as well as an corresponding
+  # response data that can later be appended on.
```

Comment:
What does "an corresponding response data that can later be appended on" refer to?

Reply:
ohhhh... this is a complete copy/paste from AuthenticationResponse. i'll come up with better wording :grimacing: